packages/e2e-test-app/test/ViewComponentTest.test.ts

```diff
@@ -49,4 +49,16 @@ describe('ViewTests', () => {
     const dump = await dumpVisualTree('backface-visibility-2');
     expect(dump).toMatchSnapshot();
   });
+  test('Views can have accessibility customization', async () => {
+    const dump = await dumpVisualTree('accessibility');
```

Comment:
Curious to know if something is wrong with accessibility, would these tests fail? or are these more so we can test manually on RNTester?

Reply:
No they wouldn't. Right now the tests just take visual snapshots! The next step is to add interactive tests to confirm that the accessibility behavior is correct.